tests/test_htmltag.py

```diff
@@ -313,12 +313,13 @@ def test_intersphinx_all_mappings(app, status, warning):
 
     chunks_regex = [
         # Python's links do have hoverxref enabled
-        r'<a class="hoverxref tooltip reference external" href="https://docs.python.org/3/tutorial/index.html#tutorial-index" title="\(in Python v3.\d\d?\)"><span class="xref std std-ref">This a :ref: to The Python Tutorial using intersphinx</span></a>',
-        r'<a class="hoverxref tooltip reference external" href="https://docs.python.org/3/library/datetime.html#datetime-datetime" title="\(in Python v3.\d\d?\)"><span class="xref std std-ref">This a :ref: to datetime.datetime Python’s function using intersphinx</span></a>',
-        r'<a class="hoverxref modal reference external" href="https://docs.python.org/3/library/functions.html#float" title="\(in Python v3.\d\d?\)"><code class="xref py py-class docutils literal notranslate"><span class="pre">float</span></code></a>',
-
+        r'<a class=\"hoverxref tooltip reference external\" href=\"https://docs.python.org/3/tutorial/index.html#tutorial-index\" title=\"\(in Python v3.\d\d?\)\"><span class=\"xref std std-ref\">This a :ref: to The Python Tutorial using intersphinx</span></a>',
+        r'<a class=\"hoverxref tooltip reference external\" href=\"https://docs.python.org/3/library/datetime.html#datetime-datetime\" title=\"\(in Python v3.\d\d?\)\"><span class=\"xref std std-ref\">This a :ref: to datetime.datetime Python function using intersphinx</span></a>',
+        r'<a class=\"hoverxref modal reference external\" href=\"https://docs.python.org/3/library/functions.html#float\" title=\"\(in Python v3.\d\d?\)\"><code class=\"xref py py-class docutils literal notranslate\"><span class=\"pre\">float</span></code></a>',
+        r'<a class=\"hoverxref tooltip reference external\" href=\"https://docs.python.org/3/library/collections.html#collections.Counter.elements\" title=\"\(in Python v3.\d\d?\)\"><code class=\"xref py py-meth docutils literal notranslate\"><span class=\"pre\">collections.Counter.elements\(\)</span></code></a> to test intersphinx methods.</p>',
+        r'<a class=\"hoverxref tooltip reference external\" href=\"https://docs.python.org/3/library/functions.html#sum\" title=\"\(in Python v3.\d\d?\)\"><code class=\"xref py py-func docutils literal notranslate\"><span class=\"pre\">sum\(\)</span></code></a> to test intersphinx functions.</p>',
         # Read the Docs' link does have hoverxref enabled
-        r'<a class="hoverxref modal reference external" href="https://docs.readthedocs.io/en/stable/config-file/v2.html#python" title="\(in Read the Docs v\d\d?.\d\d?.\d+\)"><span class="xref std std-ref">This a :ref: to Config File v2 Read the Docs’ page using intersphinx</span></a>',
+        r'<a class=\"hoverxref modal reference external\" href=\"https://docs.readthedocs.io/en/stable/config-file/v2.html#python\" title=\"\(in Read the Docs v\d\d?.\d\d?.\d+\)\"><span class=\"xref std std-ref\">This a :ref: to Config File v2 Read the Docs page using intersphinx</span></a>',
```

Comment:
These `\"` are not required. It should just work with regular `"` since the string is defined with `'`